src/Catty/DataModel/Project/Project.m

```diff
@@ -329,6 +334,29 @@ + (nullable instancetype)projectWithLoadingInfo:(ProjectLoadingInfo*)loadingInfo
     NSDebug(@"%@", [project description]);
     NSDebug(@"ProjectResolution: width/height:  %f / %f", project.header.screenWidth.floatValue, project.header.screenHeight.floatValue);
     [self updateLastModificationTimeForProjectWithName:loadingInfo.visibleName projectID:loadingInfo.projectID];
+    
+    CBFileManager *fileManager = [[CBFileManager alloc] init];
+    NSString *defaultSceneDirectoryPath = [NSString stringWithFormat:@"%@%@", [project projectPath], [Util defaultSceneNameForSceneNumber:1]];
+    if (![fileManager directoryExists:defaultSceneDirectoryPath]) {
+        project.header.catrobatLanguageVersion = Util.catrobatLanguageVersion;
+        ProjectMigrator *migrator = [[ProjectMigrator alloc] initWithFileManager:fileManager];
+        NSError *error;
+        [migrator migrateToSceneWithProject:project error:&error];
+        
+        
+        NSString *autoScreenShotFilePath = [NSString stringWithFormat:@"%@/%@", defaultSceneDirectoryPath, kScreenshotAutoFilename];
```

Comment:
Please copy/create those screenshots directly in the `ProjectMigrator`, so that they are both in the scene's folder and in the root directory. Please add some migrator tests. Thanks!